packages/apps/desktop-viewer-test/src/frontend/components/home/Recents.tsx

```diff
@@ -32,8 +37,15 @@ export const Recents = () => {
         if (displayValue.length > 25) {
           displayValue = `${displayValue.substring(0, 25)}...`;
         }
+
         return (
-          <span key={recent.path} onClick={() => openFile(recent.path)}>
+          <span
+            key={recent.path}
+            onClick={() => openFile(recent)}
+            className={!recent.path ? "disabled-link-recent" : ""}
```

Comment:
also, the class is being defined in Home.scss which you do not seem to be importing

Reply:
> instead of ternary couldnt you use `&&` ?

For classnames, it's more common to use ternary since && is a bit weird when working with classnames. Also, tried it with the !recent.path it would give an error like this Type 'false | "Deleted"' is not assignable to type 'string | undefined'.